docs/ref/models/indexes.txt

```diff
@@ -32,6 +32,19 @@ references of :class:`Index` which includes the `index options`_.
 
 A list of the name of the fields on which the index is desired.
 
+By default the indexes are created with an ascending order for all the columns.
+To define an index with a descending order for a particular column, a hyphen
+(-) can be added before the field's name while defining the index.
+
+For example::
+
+    Index(fields=['headline', '-pub_date'], name='headline_pub_date_reverse_idx')
+
+The above index would be issue an SQL with ``(headline, pub_date DESC)``. This
+is supported only for PostgreSQL, Oracle and some versions of SQLite. If your
+database doesn't support defining ordering on index columns it will simply be
```

Comment:
Are you sure that databases are just going to ignore that syntax if they don't support it? Is there a test for it?